libs/remix-ui/debugger-ui/src/lib/debugger-ui.tsx

```diff
@@ -183,8 +187,12 @@ export const DebuggerUI = (props: DebuggerUIProps) => {
       console.error(e)
     }
     let currentReceipt
+    let currentBlock
+    let currentTransaction
     try {
       currentReceipt = await web3.eth.getTransactionReceipt(txNumber)
```

Comment:
I think `txNumber` is not a good variable name in long run

Reply:
historically, this can accept blockNumber and txIndex (`txNumber`) or just txHash, but yes this is confusing.
I'd do this change in this Pr though if that's ok.